scripts/buildCSS.js

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const CleanCSS = require('clean-css');
+
+const CSSInterface = require('react-with-styles-interface-css').default;
+const registerMaxSpecificity = require('react-with-styles-interface-css').registerMaxSpecificity;
+const compileCSS = require('react-with-styles-interface-css/compile').default;
+
+const registerInterfaceWithDefaultTheme = require('../src/utils/registerInterfaceWithDefaultTheme').default;
+
+const args = process.argv.slice(2);
+const optimizeForProduction = args.includes('-o') || args.includes('--optimize');
+
+require('../test/_helpers/ignoreSVGStrings');
+
+registerMaxSpecificity(11);
```

Comment:
why is this turned up to eleven?

(no but seriously, why the magic number?)

Reply:
I should leave a comment. The number comes from the maximum number of styles passed into any `css` call in this repo